src/main/scala/strawman/collection/mutable/ListBuffer.scala

```diff
@@ -12,7 +12,7 @@ import scala.Predef.{assert, intWrapper}
 
 /** Concrete collection type: ListBuffer */
 class ListBuffer[A]
-  extends Seq[A]
+  extends GrowableSeq[A]
```

Comment:
Why do we need to separately say this is growable, buildable, and a builder?  Also, can't every mutable collection be treated as its own builder?